pyseir/inference/model_fitter.py

```diff
@@ -308,52 +339,105 @@ def _fit_seir(self, R0, t0, eps, t_break, test_fraction, hosp_fraction, log10_I_
 
         return chi2_deaths + chi2_cases + chi2_hosp
 
+    def get_posterior_estimate_eps(self, R0, eps, eps_error, plot=False):
+        """
+        Generate a posterior estimate for epsilon based on the inferred R0. This
+        is a little weird right now since we actually want a prior on Reff. So
+        in this case we use the inferred R0 to convert eps -> Reff, apply a
+        prior, and invert this transform to get back to the epsilon Max
+        A-Posteriori (MAP) estimate.
+
+        Returns
+        -------
+        posterior_map_estimate: float
+            Max A-Posteriori (MAP) estimate for epsilon.
+        """
+        R_eff = R0 * eps
+        R_eff_stdev = R0 * eps_error
+
+        x = np.linspace(0.00, 10, 1001)
+        delta_x = x[1] - x[0]
+
+        # This implements a hard lower limit of 0.98.
+        # TODO: As more data comes in, relax this.. Probably just use MCMC..
+        prior = gamma.pdf((x - 0.98) / .5, 1.3)
```

Comment:
If R eff change over time and we infer it by R0 and eps, why would we need a prior for it?

Reply:
This is essentially a prior on epsilon, but since we want to define the prior based on observations of R_eff from other sources we use this procedure.

Reply:
Got it.